WooCommerce/WooCommerceUITests/Tests/ProductsTests.swift

```diff
@@ -0,0 +1,31 @@
+import UITestsFoundation
+import XCTest
+
+final class ProductsTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        continueAfterFailure = false
+
+        let app = XCUIApplication()
+        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "-ui_testing"]
+        app.launch()
+        try LoginFlow.logInWithWPcom()
+    }
+
+    func testProductsScreenLoad() throws {
+        let products = try GetMocks.readProductsData()
+        let firstProductName = products.response.jsonBody.data[0].name
+        let firstProductPrice = products.response.jsonBody.data[0].regular_price
+        let firstProductStatus = GetMocks.init().stockStatus[products.response.jsonBody.data[0].stock_status]
+        let numberOfProducts = products.response.jsonBody.data.count
```

Comment:
If unwrapping and stock status substitution is taken care of in `readProductsData()` (as proposed in <first_comment>), you no longer need this amount of variables:

```suggestion
        let firstProductData = products[0]
```

And after that, you can pass this to `verifyProductOnProductsScreen` and `verifyProductOnSingleProductScreen` (which means passing one argument instead of three), and unwrap it in the methods themselves.

Reply:
Okay, this one I wasn't able to update, I tried to pass one parameter but it doesn't recognize the type `[ProductData]`, I looks like it's because of where the file is at, mock data and test files are on `WooCommerceUITests` and the methods are on `UITestsFoundation`. 

I'm not sure how to make it work like this, yet.

Reply:
I see what you mean. There might be a better way to do this, but what worked for me is:

1. Extract the `ProductData` struct from `MockedDataReader` to a separate file, and place it in `UITestFoundation` (when creating this new file, make sure to only keep the `UITestFoundation` target checkbox selected)
2. Make all fields in `ProductData` `public`:

<img width="439" alt="Screenshot 2021-12-17 at 15 46 12" src="https://user-images.githubusercontent.com/73365754/146553786-d1542d02-fc2d-4cce-bc00-a48f8c06ec8c.png">

3. It works!